x-pack/plugins/security_solution/public/detections/components/alerts_table/actions.tsx

```diff
@@ -150,8 +150,10 @@ export const getThresholdAggregationDataProvider = (
   ];
 };
 
-export const isEqlRule = (ecsData: Ecs) =>
-  ecsData.signal?.rule?.type?.length && ecsData.signal?.rule?.type[0] === 'eql';
+export const isEqlRuleWithGroupId = (ecsData: Ecs) =>
+  ecsData.signal?.rule?.type?.length &&
+  ecsData.signal?.rule?.type[0] === 'eql' &&
```

Comment:
We've also got those [rule type helpers](https://github.com/elastic/kibana/blob/master/x-pack/plugins/security_solution/common/detection_engine/utils.ts#L28-L32) that could be leveraged here!